src/Examples/UnitModels/Operations/feed_testing.ipynb

```diff
@@ -0,0 +1,376 @@
+      "Number of objective gradient evaluations             = 1\n",
+      "Number of equality constraint evaluations            = 1\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 1\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 0\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.001\n",
+      "Total CPU secs in NLP function evaluations           =      0.000\n",
+      "\n",
+      "EXIT: Optimal Solution Found.\n",
+      "\b\b\b\b\b\b\b\b\b\b\b\b\b\b"
+     ]
+    }
+   ],
+   "source": [
+    "# Solve the simulation using ipopt\n",
+    "# Note: If the degrees of freedom = 0, we have a square problem\n",
+    "opt = SolverFactory('ipopt')\n",
+    "result = opt.solve(m, tee=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from pyomo.opt import TerminationCondition, SolverStatus\n",
+    "\n",
+    "# Check if termination condition is optimal\n",
+    "assert result.solver.termination_condition == TerminationCondition.optimal\n",
+    "assert result.solver.status == SolverStatus.ok"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### View Results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "====================================================================================\n",
+      "Unit : fs.feed                                                             Time: 0.0\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                               Units        Outlet  \n",
+      "    flow_mol                mole / second     100.00\n",
+      "    mole_frac_comp benzene  dimensionless    0.60000\n",
+      "    mole_frac_comp toluene  dimensionless    0.40000\n",
+      "    temperature                    kelvin     298.15\n",
+      "    pressure                       pascal 1.0132e+05\n",
+      "====================================================================================\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Display output report\n",
+    "m.fs.feed.report()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import pytest\n",
+    "\n",
+    "# Check results\n",
+    "assert value(m.fs.feed.outlet.flow_mol[0]) == pytest.approx(100, abs=1e-2)\n",
+    "assert value(m.fs.feed.mole_frac_comp[0, \"benzene\"]) == pytest.approx(0.6, abs=1e-2)\n",
```

Comment:
1. Relative tolerances are almost always better than absolute tolerances (unless you are dealing with very small numbers).
2. 1e-2 is very loose tolerance, especially given that this is virtually a no-op unit (i.e. we should expect very high tolerance).